flygym/preprogrammed.py

```diff
@@ -96,3 +97,53 @@ def get_collision_geometries(config: str = "all") -> List[str]:
         return []
     else:
         raise ValueError(f"Unknown collision geometry configuration: {config}")
+
+
+def get_cpg_biases(gait: str) -> np.ndarray:
```

Comment:
I added the hardcoded CPG phase biases to `preprogrammed.py`. It's nice if we use this as the single place where these are coded (call this function instead of defining them again).